webui/react/src/stores/permissions.tsx

```diff
@@ -1,47 +1,45 @@
 import { Loadable, Loaded, NotLoaded } from 'hew/utils/loadable';
```

Comment:
@mapmeld Can you review this file?

Reply:
OK I checked permissions were working as expected on OSS (Model Registry) and EE (adding and viewing members of workspaces as admin or general user)